WordPress/Classes/Services/CommentService.m

```diff
@@ -104,6 +107,50 @@ - (void)syncCommentsForBlog:(Blog *)blog
                        }];
 }
 
+- (void)loadMoreCommentsForBlog:(Blog *)blog
+                        success:(void (^)(BOOL hasMore))success
+                        failure:(void (^)(NSError *))failure
+{
+    id<CommentServiceRemote> remote = [self remoteForBlog:blog];
+    NSMutableDictionary *options = [NSMutableDictionary dictionary];
+    if ([remote isKindOfClass:[CommentServiceRemoteREST class]]) {
+        NSString *entityName = NSStringFromClass([Comment class]);
+        NSFetchRequest *request = [NSFetchRequest fetchRequestWithEntityName:entityName];
+        request.predicate = [NSPredicate predicateWithFormat:@"dateCreated != NULL && blog=%@", blog];
+        NSSortDescriptor *sortDescriptor = [NSSortDescriptor sortDescriptorWithKey:@"dateCreated" ascending:YES];
+        request.sortDescriptors = @[sortDescriptor];
+        Comment *oldestComment = [[self.managedObjectContext executeFetchRequest:request error:nil] firstObject];
+        if (oldestComment.dateCreated) {
+            options[@"before"] = [oldestComment.dateCreated WordPressComJSONString];
+            options[@"order"] = @"desc";
+        }
+    } else if ([remote isKindOfClass:[CommentServiceRemoteXMLRPC class]]) {
+        NSUInteger postCount = [blog.posts count];
+        options[@"offset"] = @(postCount);
```

Comment:
Did you mean `blog.comments`? :wink: 
Also, this would miss comments if some were remotely deleted since the last sync:
- Let's say there's 102 comments
- We initially sync 100
- We trash 2 on the server, there are 100 left
- We scroll down, offset is 100, 0 comments are returned